lib/house.rb

```diff
@@ -0,0 +1,27 @@
+class House
+  attr_reader :room,
+              :price,
+              :address,
+              :rooms
+
+
+  def initialize(price, address)
+    @price   = (price.delete("$").to_i)
+    @address = address
+    @rooms  = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if price < 500000
+      false
+    end
```

Comment:
This works for houses that are below market average, but if a house is more that 500K, this method will return `nil`; it makes more sense if it returned `true` in that case.  Either of the below options would work:

```ruby
if price < 500000
  false
else
  true
end
```
OR
```
price < 500000
```